src/test/java/edu/harvard/iq/dataverse/api/FilesIT.java

```diff
@@ -2483,4 +2485,122 @@ public void testCollectionStorageQuotas() {
         
         UtilIT.deleteSetting(SettingsServiceBean.Key.UseStorageQuotas);
     }
+
+    @Test
+    public void testFileCitation() throws IOException {
+        Response createUser = UtilIT.createRandomUser();
+        createUser.then().assertThat().statusCode(OK.getStatusCode());
+        String apiToken = UtilIT.getApiTokenFromResponse(createUser);
+
+        Response createDataverseResponse = UtilIT.createRandomDataverse(apiToken);
+        createDataverseResponse.then().assertThat().statusCode(CREATED.getStatusCode());
+        String dataverseAlias = UtilIT.getAliasFromResponse(createDataverseResponse);
+
+        Response createDatasetResponse = UtilIT.createRandomDatasetViaNativeApi(dataverseAlias, apiToken);
+        createDatasetResponse.prettyPrint();
+        createDatasetResponse.then().assertThat().statusCode(CREATED.getStatusCode());
+        Integer datasetId = JsonPath.from(createDatasetResponse.body().asString()).getInt("data.id");
+        String datasetPid = JsonPath.from(createDatasetResponse.body().asString()).getString("data.persistentId");
+
+        Response getDatasetVersionCitationResponse = UtilIT.getDatasetVersionCitation(datasetId, DS_VERSION_DRAFT, false, apiToken);
```

Comment:
I would remove this call with the associated assertion to make the test shorter and more specific to the method we are testing.

This is already tested on `getDatasetVersionCitation` test.

Reply:
Right, right, I was comparing methods, trying to make them similar. I'll probably remove it. Thanks.

Reply:
Done as part of this: e9ab40bdaa simplify and rename tests